<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>128</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>128</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QWidget {
    font: 580 10pt &quot;Segoe UI&quot;;
    border: 2px solid black;
    border-radius: 0px;
    border-left: 0px;
    border-right: 0px;
	border-top: 0px;
    border-color: rgba(50, 50, 50, 255);
    padding: 8px;
    padding-left: 6px;
    padding-right: 6px;
    color: #E3E3E3;
    background-color: #1C1C1C;
}

.QWidget::hover {
}
.QWidget::selected {
    background-color: #414956;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">.QFrame {
    font: 580 10pt &quot;Segoe UI&quot;;
    border: 2px solid black;
    border-radius: 0px;
    border-left: 0px;
    border-right: 0px;
	border-top: 0px;
    border-color: rgba(50, 50, 50, 255);
    color: #E3E3E3;
    background-color: #1C1C1C;
}

.QFrame::hover {
}
.QFrame::selected {
    background-color: #414956;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">border:0px;
background-color: rgba(255, 255, 255, 0);
font: 8pt &quot;Segoe UI&quot;;
color: #ECA4A0;
</string>
        </property>
        <property name="text">
         <string>Vector X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="vector_x">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QDoubleSpinBox {
    font: 580 8pt &quot;Segoe UI&quot;;
    border: 2px solid black;
    border-radius: 2px;
    border-color: rgba(80, 80, 80, 255);
    color: #E3E3E3;
    background-color: #1C1C1C;
border:0px;
color: #ECA4A0;
}

QDoubleSpinBox:focus {
}

QDoubleSpinBox:hover {
}

QDoubleSpinBox:pressed {
	background-color: #121212;
}

QDoubleSpinBox::up-button {
    border: 0px solid black;
    subcontrol-origin: border;
    subcontrol-position: top right;
    border-left: 2px solid black;
    border-bottom: 0px solid black;
    border-top-right-radius: 2px;
    border-color: rgba(80, 80, 80, 255);
	width:16px;
    margin:0px;
border:0px;
}

QDoubleSpinBox::up-arrow {
    image: url(://icons/arrow_drop_up_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg);
    width: 20px;
    height: 20px;
}

QDoubleSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position: bottom right;
	border: 0px solid black;
    border-left: 2px solid black;
    border-top: 2px solid black;
border-bottom-right-radius: 2px;
    border-color: rgba(80, 80, 80, 255);
	width:16px;
    margin:0px;
border:0px;
}

QDoubleSpinBox::down-arrow {
    image: url(://icons/arrow_drop_down_16dp.svg);
    width: 20px;
    height: 20px;
}</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>-99999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99999999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">.QFrame {
    font: 580 10pt &quot;Segoe UI&quot;;
    border: 2px solid black;
    border-radius: 0px;
    border-left: 0px;
    border-right: 0px;
	border-top: 0px;
    border-color: rgba(50, 50, 50, 255);
    color: #E3E3E3;
    background-color: #1C1C1C;
}

.QFrame::hover {
}
.QFrame::selected {
    background-color: #414956;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">border:0px;
background-color: rgba(255, 255, 255, 0);
font: 8pt &quot;Segoe UI&quot;;
color: #B6EFA2;</string>
        </property>
        <property name="text">
         <string>Vector Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="vector_y">
        <property name="styleSheet">
         <string notr="true">QDoubleSpinBox {
    font: 580 8pt &quot;Segoe UI&quot;;
    border: 2px solid black;
    border-radius: 2px;
    border-color: rgba(80, 80, 80, 255);
    color: #E3E3E3;
    background-color: #1C1C1C;
border:0px;
color: #B6EFA2;
}

QDoubleSpinBox:focus {
}

QDoubleSpinBox:hover {
}

QDoubleSpinBox:pressed {
	background-color: #121212;
}

QDoubleSpinBox::up-button {
    border: 0px solid black;
    subcontrol-origin: border;
    subcontrol-position: top right;
    border-left: 2px solid black;
    border-bottom: 0px solid black;
    border-top-right-radius: 2px;
    border-color: rgba(80, 80, 80, 255);
	width:16px;
    margin:0px;
border:0px;
}

QDoubleSpinBox::up-arrow {
    image: url(://icons/arrow_drop_up_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg);
    width: 20px;
    height: 20px;
}

QDoubleSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position: bottom right;
	border: 0px solid black;
    border-left: 2px solid black;
    border-top: 2px solid black;
border-bottom-right-radius: 2px;
    border-color: rgba(80, 80, 80, 255);
	width:16px;
    margin:0px;
border:0px;
}

QDoubleSpinBox::down-arrow {
    image: url(://icons/arrow_drop_down_16dp.svg);
    width: 20px;
    height: 20px;
}

QDoubleSpinBox:disabled {
	color: rgba(125, 125, 125, 100);
}</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>-99999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99999999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">.QFrame {
    font: 580 10pt &quot;Segoe UI&quot;;
    border: 2px solid black;
    border-radius: 0px;
    border-left: 0px;
    border-right: 0px;
	border-top: 0px;
    border-color: rgba(50, 50, 50, 255);
    color: #E3E3E3;
    background-color: #1C1C1C;
}

.QFrame::hover {
}
.QFrame::selected {
    background-color: #414956;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">border:0px;
background-color: rgba(255, 255, 255, 0);
font: 8pt &quot;Segoe UI&quot;;

color: #A4B6EF;</string>
        </property>
        <property name="text">
         <string>Vector Z</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="vector_z">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QDoubleSpinBox {
    font: 580 8pt &quot;Segoe UI&quot;;
    border: 2px solid black;
    border-radius: 2px;
    border-color: rgba(80, 80, 80, 255);
    color: #E3E3E3;
    background-color: #1C1C1C;
border:0px;
color: #A4B6EF;
}

QDoubleSpinBox:focus {
}

QDoubleSpinBox:hover {
}

QDoubleSpinBox:pressed {
	background-color: #121212;
}

QDoubleSpinBox::up-button {
    border: 0px solid black;
    subcontrol-origin: border;
    subcontrol-position: top right;
    border-left: 2px solid black;
    border-bottom: 0px solid black;
    border-top-right-radius: 2px;
    border-color: rgba(80, 80, 80, 255);
	width:16px;
    margin:0px;
border:0px;
}

QDoubleSpinBox::up-arrow {
    image: url(://icons/arrow_drop_up_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg);
    width: 20px;
    height: 20px;
}

QDoubleSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position: bottom right;
	border: 0px solid black;
    border-left: 2px solid black;
    border-top: 2px solid black;
border-bottom-right-radius: 2px;
    border-color: rgba(80, 80, 80, 255);
	width:16px;
    margin:0px;
border:0px;
}

QDoubleSpinBox::down-arrow {
    image: url(://icons/arrow_drop_down_16dp.svg);
    width: 20px;
    height: 20px;
}

QDoubleSpinBox:disabled {
	color: rgba(125, 125, 125, 100);
}</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>-99999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99999999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">.QFrame {
    font: 580 10pt &quot;Segoe UI&quot;;
    border: 2px solid black;
    border-radius: 0px;
    border-left: 0px;
    border-right: 0px;
	border-top: 0px;
    border-color: rgba(50, 50, 50, 255);
    color: #E3E3E3;
    background-color: #1C1C1C;
}

.QFrame::hover {
}
.QFrame::selected {
    background-color: #414956;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">border:0px;
background-color: rgba(255, 255, 255, 0);
font: 8pt &quot;Segoe UI&quot;;

color: #EFCBA0;</string>
        </property>
        <property name="text">
         <string>Vector W</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="vector_w">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QDoubleSpinBox {
    font: 580 8pt &quot;Segoe UI&quot;;
    border: 2px solid black;
    border-radius: 2px;
    border-color: rgba(80, 80, 80, 255);
    color: #E3E3E3;
    background-color: #1C1C1C;
border:0px;
color: #EFCBA0;
}

QDoubleSpinBox:focus {
}

QDoubleSpinBox:hover {
}

QDoubleSpinBox:pressed {
	background-color: #121212;
}

QDoubleSpinBox::up-button {
    border: 0px solid black;
    subcontrol-origin: border;
    subcontrol-position: top right;
    border-left: 2px solid black;
    border-bottom: 0px solid black;
    border-top-right-radius: 2px;
    border-color: rgba(80, 80, 80, 255);
	width:16px;
    margin:0px;
border:0px;
}

QDoubleSpinBox::up-arrow {
    image: url(://icons/arrow_drop_up_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg);
    width: 20px;
    height: 20px;
}

QDoubleSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position: bottom right;
	border: 0px solid black;
    border-left: 2px solid black;
    border-top: 2px solid black;
border-bottom-right-radius: 2px;
    border-color: rgba(80, 80, 80, 255);
	width:16px;
    margin:0px;
border:0px;
}

QDoubleSpinBox::down-arrow {
    image: url(://icons/arrow_drop_down_16dp.svg);
    width: 20px;
    height: 20px;
}

QDoubleSpinBox:disabled {
	color: rgba(125, 125, 125, 100);
}</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>-99999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99999999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
